public/app/plugins/datasource/elasticsearch/components/QueryEditor/BucketAggregationsEditor/utils.ts

```diff
@@ -123,6 +124,8 @@ const INCOMPATIBLE_ORDER_BY_AGGS = ['top_metrics'];
 export const createOrderByOptionsFromMetrics = (metrics: MetricAggregation[] = []): OrderByOption[] => {
   const metricOptions = metrics
     .filter((metric) => !INCOMPATIBLE_ORDER_BY_AGGS.includes(metric.type))
+    // pipeline aggregations can't be used for sorting: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html#search-aggregations-bucket-terms-aggregation-order
```

Comment:
nitpick: It seems like both filters do similar thing (removing incompatible metrics), it could read a bit better if it was a single `.filter()` callback.

Reply:
i actually used a different filter because I thought it read better 🤔 

btw, i should add some tests, i wanted to create a draft PR but clicked on open without thinking :D